integration_tests/mockApis/nomisMigrationApi.ts

```diff
@@ -214,10 +214,170 @@ const stubGetFailures = (failures: unknown = defaultFailures): SuperAgentRequest
     },
   })
 
+const stubGetMigrationDetailsStarted = (migrationId: string): SuperAgentRequest =>
+  stubFor({
```

Comment:
DPS has a [getFor](https://github.com/ministryofjustice/digital-prison-services/blob/c6a8c5825780edfdb1d42e364a500eb9923d04eb/integration-tests/mockApis/wiremock.js#L46) that reduces some of this boiler plate